bind/motor-mobile/object_builder.go

```diff
@@ -1,34 +1,20 @@
 package motor
 
 import (
-	"errors"
 	"fmt"
 
 	_ "golang.org/x/mobile/bind"
 )
 
-/*
-#include <stdlib.h>
-*/
-import "C"
-
-func NewObjectBuilder(n, d *C.char) error {
+func NewObjectBuilder(name, schemaDid string) error {
 	if instance == nil {
 		return errWalletNotExists
 	}
-	if n == nil {
-		return errors.New("name cannot be nil")
-	}
-	if d == nil {
-		return errors.New("schema did cannot be nil")
-	}
 
-	name := C.GoString(n)
 	if _, ok := objectBuilders[name]; ok {
```

Comment:
I don't see objectBuilders coming from anywhere either? Are these automatically sourced from bind or somewhere else in the package?

Reply:
Yes, for this and the above, look in `motor-mobile/bind.go`

```

var (
	instance       mtr.MotorNode
	objectBuilders map[string]*object.ObjectBuilder
	callback       MotorCallback
)
```